docs/11.EXT-REFERENCE-MODULE.md

```diff
@@ -0,0 +1,108 @@
+#  Module API
+
+## jerryx_module_resolve
+
+**Summary**
+
+Load a copy of a module into the current context or return one that was already loaded if it is found.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerryx_module_resolve (const jerry_char_t *name, /**< name of the module to load */
+                       jerryx_module_resolver_t *resolvers_p, /**< list of resolvers */
+					   size_t resolver_count); /**< number of resolvers in @p resolvers */
+```
+
+- `name` - the name of the module to load
+- `resolvers_p` - the list of resolvers to call
+- `resolver_count` - the number of resolvers in `resolvers_p`
+- return value - `jerry_value_t` representing the module that was loaded, or the error that occurred in the process.
+
+
+## jerryx_module_native_resolver
+
+**Summary**
+
+The resolver for JerryScript modules. A pointer to this function can be passed in the second parameter to `jerryx_module_resolve` to search for the module among the JerryScript modules built into the binary.
```

Comment:
@jiangzidong the introduction at the top of the file now covers both these topics.